sdk/ml/azure-ai-ml/azure/ai/ml/operations/_job_operations.py

```diff
@@ -176,6 +177,7 @@ def __init__(
             self._all_operations, self._operation_scope, self._operation_config
         )  # pylint: disable=line-too-long
 
+        self.service_client_08_2023_preview = kwargs.pop("service_client_08_2023_preview", None)
```

Comment:
would it be better to add this as a named argument? why do we need to hide it in kwargs?

Reply:
Based on my understanding of this function, I believe that only necessary parameters are disclosed, while other unnecessary parameters are hidden.
However, if this parameter is written as public, I think there is no problem.

![image](https://github.com/Azure/azure-sdk-for-python/assets/23182548/7e20c6a1-4744-45f5-82d1-2e69e37bc952)
![image](https://github.com/Azure/azure-sdk-for-python/assets/23182548/a6babad2-9cef-4057-9f6b-227463cf5d94)

Reply:
done